README.md

```diff
@@ -776,6 +776,8 @@ scan:
                 [-ask-key-password]
                 [-debug]
                 [-pipe]
+                [-timeout]
+                [-timeout-detect-platform]
```

Comment:
I understand what you mean.
However, `timeout` option is required not only for `DetectPlatforms` but also for `InitServers`.
So, we have three choices.
1. `-timeout(for scan)` , `-timeout-detect-platform` , `-timeout-init-servers`
2. `-timeout(for scan)` , `-timeout-except-scan`(for `DetectPlatforms` and `InitServers`)
3. `-timeout`(for `DetectPlatforms` and `InitServers`), `-timeout-scan(for scan)`

I think No.3 is good, because `InitServers` is called in `configtest`.
It seems to be better to align the option name to `-timeout` with `configtest` and `scan`.
https://github.com/future-architect/vuls/blob/33b2aa2d52c0b9954f45d7769b37954bf4361802/commands/configtest.go#L167

Do you have any other good idea?